src/puppetlabs/dujour/version_check.clj

```diff
@@ -118,7 +128,21 @@
   ([request-values update-server-url callback-fn]
     (validate-config! request-values update-server-url)
     (future
-      (let [server-response (try
+      (let [certname (:certname request-values)
+            request-values (if-not (nil? certname)
+                             (->
+                               request-values
+                               (assoc :host-id (get-hash certname))
+                               (dissoc :certname))
+                             request-values)
+            cacert (:cacert request-values)
+            request-values (if-not (nil? cacert)
+                             (->
+                               request-values
+                               (assoc :site-id (get-hash cacert))
+                               (dissoc :cacert))
+                             request-values)
```

Comment:
I think this might not be required if we rename the original request values to be site-id and host-id...right?